tox.ini

```diff
@@ -27,11 +27,10 @@ skip_glob=
 
 [testenv]
 deps =
-passenv = CI TRAVIS TRAVIS_* GOPATH
+passenv = CI TRAVIS TRAVIS_*
 extras = test
 commands =
-    test: py.test --ignore=tests/interop
-    interop: py.test tests/interop
+    py.test tests/
```

Comment:
I can maybe imagine this command accidentally being run in future test environments that we didn't intend. One option is to re-add the `test:` prefix.